lib/game.py

```diff
@@ -69,23 +70,24 @@ def periodic(self):
     return
 
   def show(self,wait=0.3):
-    TEXT = '\033[38;2;'
-    COLOUR_TEXT = "{r};{g};{b}m"
-    WHITE_TEXT = TEXT + '255;255;255m'
-
-    print(chr(27) + "[2J")
-    print('\033[H')
-    print (WHITE_TEXT + 'At tick number ' + str(self.ticks))
+    i = 0
+    j = 0
+    self.stdscr.addstr(i, 0, 'At tick number ' + str(self.ticks), curses.color_pair(1))
```

Comment:
DISRESPECTS the rgb in the rule.colour dict, but i will fix!

Reply:
consider urself disrespected ohohoh. Also change the things in `color_pair` in `load_game` to fix this easily. Can just init a bunch of pairs from your dict.